snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -465,11 +456,11 @@
     // Listen for clicks on links, send metrics and handle
     // about:account custom links.
     var topSection = document.getElementById('topSection');
-    topSection.addEventListener('click', function(event) {
+    function trackLinkClicks(event) {
         var target = event.target;
         while (target.tagName && target.tagName.toLowerCase() !== 'a') {
             // Do not track clicks outside topSection.
-            if (target.id === 'topSection') {
+            if (target.id === 'topSection' || !target.parentNode) {
```

Comment:
If we don't have a `parentNode`, we're no longer in the DOM and thus can return.